### ISSUE TITLE
[Feature Request] Sleep after xx

### ISSUE THREAD

**Issue by C9Glax** [CappielloAntonio/tempo#68]
The feature would allow the user to configure a timeout, after which the music stops. The menu could be located on the playback-screen (top-right, maybe burger menu?)

Options could be

- After Track
- After 10 Minutes
- After 22 o'clock

To further enhance, the stop could even be a fade-out (60 seconds or so).

**Comment by alehmann76** [comment]
I have an addition to this requests:
A  countdown timer with an customizable alarm sound. The reason is that i enjoy small power naps for about 25min. And when i use my bluetooth Headphones with anc i dont hear my Alarm clock! 
Just an Idea... 😁

**Comment by eddyizm** [comment]
hi @C9Glax @alehmann76 
My fork has continued developement and a Sleep timer has been implemented in the latest release [v4.17.0](https://github.com/eddyizm/tempus/releases/tag/v4.17.0)

Feel free to close this issue as this repo has been abandoned.

**Comment by C9Glax** [comment]
@eddyizm just checked out the update, been using it for a long time now and cool to see you still checking of requests here 👍🏼

**Comment by eddyizm** [comment]
> [@eddyizm](https://github.com/eddyizm) just checked out the update, been using it for a long time now and cool to see you still checking of requests here 👍🏼
Yeah, i really started because the app had all the same bugs, obvs, lol, and I forked it because I didn't want to go to another app and see this one die. So hoping others still know there is an option. 

It looks like the sleep feature has a small bug but we are aware of it and another dev is going to apply the fix soon.  Thanks for closing it here though. @C9Glax
